### ISSUE TITLE
Relation between Vendor(Supplier) and Items

### ISSUE THREAD

**Issue by yadgar222** [jekkos/opensourcepos#1145]
Hi,
There is no Relation Between Supplier And Items, I mean when clicking on Supplier doesn't show your the Items (Products) that they supply and there balance also

**Comment by daN4cat** [comment]
@yadgar222 yes, good proposal for an addition. Feel free to contribute it, we need help.
